src/PEImage.cpp

```diff
@@ -35,19 +35,6 @@ PEImage::PEImage(const TCHAR* iname)
 , hdr32(0)
 , hdr64(0)
 , fd(-1)
-, debug_aranges(0)
-, debug_pubnames(0)
-, debug_pubtypes(0)
-, debug_info(0), debug_info_length(0)
-, debug_abbrev(0), debug_abbrev_length(0)
-, debug_line(0), debug_line_length(0)
-, debug_frame(0), debug_frame_length(0)
-, debug_str(0)
-, debug_loc(0), debug_loc_length(0)
-, debug_ranges(0), debug_ranges_length(0)
-, codeSegment(0)
-, linesSegment(-1)
-, reloc(0), reloc_length(0)

```

Comment:
As mentioned in https://github.com/rainers/cv2pdb/pull/68#pullrequestreview-818909474, I like to review PRs commit by commit, and if there is too much in one single commit, it is easy for bugs to hide. I did not find any bug in this, but I would not be surprised if I missed any because of the different goals of this commit: dropping `debug_aranges` and friends, refactoring `PESection`s, adding new ones, dropping commented-out code, adding debugging, etc ;-)

Reply:
I understand your concern.  I was mainly aiming here for a testable chunk which shouldn't introduce functional changes.

I have a different philosophy about code review: it's really only worthwhile for architectural analysis and for suggesting ways to improve the readability or understandability of code.  For bugs, however, tests are the gold standard.  Especially for code like this where an off-by-one error or subtle spec misreading can lead to wildly different results.  I think we've seen over the decades that the "thousand eyes make bugs shallow" idea doesn't really work in practice.